### ISSUE TITLE
P_miss Function

### ISSUE THREAD

**Issue by Robo-NA** [gongziyida/Infotaxis#3]
Hello. Thanks for the code, but I have a question about the function P_miss. From what I understand, you're calculating the probability of getting a miss in a time duration. From the Infotaxis paper, it is calculated by substituting in rho_0, where you end up with rho_0 = exp(-dt * average_num_hits). But in your code, you average the exp(-dt * hit_rate) over all source locations with sum(exp(-dt * hit_rate) * probability_distribution).
My question is, why not get this by averaging the hit_rate, resulting in rho_0 = exp(-dt*sum(hit_rate * probability_distribution))? Wouldn't that be closer to what the paper is saying?

**Comment by gongziyida** [comment]
Hi, thank you for your interest. This is a nice catch! It has been a long time since I programmed this, and I guess I understood the P_miss part as conditional: P_miss = P(miss | r0) * P(r0 | Trajectory), where the paper calculates it as P_miss ~ Poisson(mean rate).

I think due to exp is monotonic, the difference should not affect the result qualitatively (let me know if I am wrong).

I am still not sure why the authors do not choose the alternative way, though. Maybe it is related to physics principles?

**Comment by Robo-NA** [comment]
Hmm, would it be correct to say, in layman's terms not mathematical, in your code "Probability of missing the source = probability of getting a miss given the source location averaged over all source positions."
And for the paper, "Probability of missing the source is a poisson distribution of the average number of hits over all source positions."
